### PR TITLE
Form Components now have default values for all fields

### PR DESCRIPTION
**Pull request by mjashanks**
Fixes for #477 and #476 

Select fields default to the first option

### CODE REVIEW

File: packages/standard-components/src/DataForm.svelte [hunk 1]
```diff
@@ -32,11 +32,35 @@
 
   $: Object.values(inputElements).length && setForm(record)
 
+  const createBlankRecord = () => {
```
Comment (shogunpurple) [inline packages/standard-components/src/DataForm.svelte:35]: Is it possible to achieve the same result with HTML5 validation on the inputs or any other kind of simpler client side validation? Ie. adding a `required` attribute? It seems like this is a hacky way to provide valid data when we should just have some validation in the client side based on the type

Comment (mjashanks) [reply packages/standard-components/src/DataForm.svelte:35]: That doesnt work for checkboxes...
- Bound field is `undefined`
- Checkbox is unchecked
- user wants it to remain unchecked
- user clicks save
- Validation error appears to flag `undefined` value
- User has to check and uncheck checkbox

Comment (shogunpurple) [reply packages/standard-components/src/DataForm.svelte:35]: Ah, gotcha. Seems like we should just allow `undefined` in the backend then to have the same meaning as `false`

Comment (mjashanks) [reply packages/standard-components/src/DataForm.svelte:35]: @shogunpurple I'm going to push you on this... I have thought about it the options above.

I feel like a new record should take a default state - with all the fields defined. 

I do think there is a better place to do this. We used to have a "CreateNew" function, which setup a new model, with all fields defined, and even had the ability to declare default values via that builder.

Aside from having that broader solution, which I think we should have, this is an easy interim solution, that dooesnt require us having to put various bits of code in different places in the application.

**Pull request merged**